src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/netcore/TensorPrimitives.PopCount.cs

```diff
@@ -13,6 +13,20 @@ namespace System.Numerics.Tensors
 {
     public static partial class TensorPrimitives
     {
+        /// <summary>Computes the population count of all elements in the specified tensor.</summary>
+        /// <param name="x">The tensor, represented as a span.</param>
+        /// <returns>The sum of the number of bits set in each element in <paramref name="x"/>.</returns>
+        public static long PopCount<T>(ReadOnlySpan<T> x) where T : IBinaryInteger<T>
+        {
+            long count = 0;
+            for (int i = 0; i < x.Length; i++)
+            {
```

Comment:
Is there a reason why this particular routine isn't being vectorized? Limited perf gains?

Reply:
It can be done later if we find wins. I'd previously tried implementing https://arxiv.org/pdf/1611.07612 and found limited gains.